guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -7,43 +7,47 @@ functional_areas:
 ---
 An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
 
-```
+```bash
```

Comment:
```suggestion
```terminal
```

Reply:
Use "bash" for actual commands and "terminal" for console output.

Reply:
Thank you.  first time i've seen that linting error.  corrected and committed 2da31df